modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -7,50 +7,50 @@ The Try scope allows you to handle errors that may occur when attempting to exec
+If an error is raised by a component inside a Try scope, then the Try scope's error handler is executed, and the error is routed to the matching handler. At this point, the error is available for inspection, so the handlers can execute and act accordingly:
 
-* An On Error Continue will execute and use the result of the execution as the result of its owner, as if the owner actually completed the execution successfully. Any transactions at this point are also committed.
-* An On Error Propagate will roll back any transactions, execute, and use that result to rethrow the existing error, meaning that its owner will be considered as failing.
+* An On Error Continue executes and sends the result of the execution to its container Try scope, which uses that result to complete the execution successfully. Any transactions at this point are also committed.
+* An On Error Propagate rolls back any transactions, then executes and uses that result to re-throw the existing error, meaning that its container Try Scope's execution fails.
 
-If the Try scope has several components, then once a component raises an exception, subsequent components in the Try scope are skipped, regardless of the type of error handler that catches the exception. In the case of On Error Propagate, the error is propagated to the flow's error handling, as if the Try scope did not exist. In the case of On Error Continue, processing continues outside the Try scope at the next downstream flow component, as if the Try scope never threw an exception.
+If the Try scope has several components, then once a component raises an exception, subsequent components in the Try scope are skipped, regardless of the type of error handler that catches the exception. In the case of On Error Propagate, the error is propagated to the flow's error handler, as if the Try scope did not exist. In the case of On Error Continue, processing continues outside the Try scope at the next downstream flow component, as if the Try scope never threw an exception.
 
-In the example above, any database connection errors (DB:CONNECTIVITY) are propagated because of the On Error Propagate (`on-error-propagate`) setting. Propagation of this error causes the Try process to fail and the flow’s error handler to execute. Other errors are handled through On Error Continue (`on-error-continue`), so the Try is treated as successful when they occur, meaning that the next processor, an HTTP request, continues to execute.
+In the example above, any database connection errors (`DB:CONNECTIVITY`) are propagated because of the On Error Propagate (`on-error-propagate`) setting. Propagation of this error causes the Try process to fail and the flow’s error handler to execute. Other errors are handled through On Error Continue (`on-error-continue`), so the Try is treated as successful when they occur, meaning that the next processor, an HTTP request, continues its execution.
```

Comment:
In the example above --> In the previous example
Since you are using in other paragraphs: In the following example. Therefore, instead of using above/below use previous/following.